test/Microsoft.AspNetCore.Sockets.Common.Tests/MessageFormatterTests.Text.cs

```diff
@@ -9,10 +9,10 @@
 
 namespace Microsoft.AspNetCore.Sockets.Tests
 {
-    public class MessageFormatterTests
+    public partial class MessageFormatterTests
```

Comment:
Not a big fan. I think it would be better to have TextFormatMessageFormatterTests and BinaryFormatMessageFormatterTests. I would not change the test names though - the prefix will make it much easier to filter tests in the test explorer (when it happens to work)